cli/flags.rs

```diff
@@ -803,6 +807,14 @@ Ignore formatting a file by adding an ignore comment at the top of the file:
         .help("Check if the source files are formatted")
         .takes_value(false),
     )
+    .arg(
+      Arg::with_name("ext")
+        .long("ext")
+        .help("Set standard input (stdin) content type")
+        .takes_value(true)
+        .default_value("ts")
```

Comment:
Wasn't the default `tsx` previously?

Reply:
I'm not sure if there's a default before, but all extensions (ts, tsx, js, jsx) are processed using the same plugin (`dprint-plugin-typescript`) - so we can use any one of them as default.